exercises/practice/forth/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 pub type Value = i32;
-pub type ForthResult = Result<(), Error>;
```

Comment:
curious: `ForthResult` feels repetitive? As in, since we are in the `forth` directory's lib.rs, it is clear from context.

Reply:
Precisely: it's stuttering to use `foo::FooThing`; we shouldn't encourage that habit.